File: abc/abc223/a.java
// https://atcoder.jp/contests/abc223/submissions/40778009

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);
    
    int x = stdIn.nextInt();

    if(x % 100 == 0 && x != 0) System.out.println("Yes");
    else System.out.println("No");
  }
}

File: abc/abc223/a.cs
// https://atcoder.jp/contests/abc223/submissions/46503028

using System;

public class A
{
  static void Main()
  {
    int x = int.Parse(Console.ReadLine());
    if(x % 100 == 0 && x != 0)
    {
      Console.WriteLine("Yes");
    }
    else
    {
      Console.WriteLine("No");
    }
  }
}

